privacyidea/lib/tokens/pushtoken.py

```diff
@@ -83,8 +83,11 @@
 POLL_TIME_WINDOW = 1
 UPDATE_FB_TOKEN_WINDOW = 5
 POLL_ONLY = "poll only"
-AVAILABLE_PRESENCE_OPTIONS = ["A", "B", "C"]
-
+AVAILABLE_PRESENCE_OPTIONS_ALPHABETIC = ["A", "B", "C", "D", "E", "F", "G", "H", "I",
+                                         "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z",]
+AVAILABLE_PRESENCE_OPTIONS_NUMERIC = ["01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12",
```

Comment:
since we need strings we could use:
```AVAILABLE_PRESENCE_OPTIONS_NUMERIC = [f"{x}" for x in string.ascii_uppercase]```
for the characters and
```AVAILABLE_PRESENCE_OPTIONS_NUMERIC = [f"{x:02}" for x in range(1, 100)]```

Reply:
or that, just don't write it out ;)